.github/workflows/persistent-devnet.yml

```
@@ -7,6 +7,9 @@ on:
         description: "Release tag, branch name or any revision."
         required: true
         type: string
+  push:
+    tags:
+      - release-v[0-9].[0-9]
```

We want the tags to be on patch versions, also removing "release" prefix

```suggestion
      - v[0-9].[0-9].[0-9]
```
Eg: v2.4.1